theme/_uswds-theme-custom-styles.scss

```diff
@@ -174,11 +174,34 @@ $spinner-border-width-sm: 0.2em;
   span {
     @include h4;
     color: color($theme-color-base-light);
-    display: block;
-    position: absolute;
+    display: grid;
   }
 }
 
+.ringdown--RINGDOWNCONFIRMED,
+.ringdown--RINGDOWNRECEIVED,
+.ringdown--OFFLOADED {
+  background-color: #ffecba;
+  color: #df7800;
+  padding: 0px 6px 0px 6px;
+  border-radius: 4px;
+}
+
+.ringdown--ARRIVED {
+  background-color: #d5fec7;
+  color: #007922;
+  padding: 0px 6px 0px 6px;
+  border-radius: 4px;
+}
+
+.margin-top-2 {
```

Comment:
It's really not appropriate to override a generic utility class like this with other styles specific to one case. You've actually just broken other places in the code that use margin-top-2 to add margin spacing.